3rdparty/mbedtls/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
+set(OE_SPECTRE_MITIGATION_FLAGS "-mllvm;-x86-speculative-load-hardening")
```

Comment:
This shouldn't exist, this variable should only be set in one place which is `compiler_settings.cmake`.

Reply:
OE_SPECTRE_MITIGATION_FLAGS shows as an uninitialised error warning . 
This error was recently introduced this week , after the fix with uninitialised variables
https://oe-jenkins.eastus.cloudapp.azure.com/blue/organizations/jenkins/Bors/detail/trying/814/pipeline